docs/eigenda/integrations-guides/dispersal/dispersal-api/overview.md

```diff
@@ -0,0 +1,166 @@
+
+The source of truth for the Disperser API spec is [disperser.proto](https://github.com/Layr-Labs/eigenda/blob/8ec570b8c2b266fad20ea0af14f0f5d84906c39c/api/proto/disperser/disperser.proto), adjusted to the current release. The goal of this document is to explain this spec at a higher level.
+
+<!-- TODO: Update to point to master, not a specific commit -->
+
+Eigen Labs hosts one disperser endpoint for each EigenDA network. These endpoints are documented in respective network pages [indexed here](../../../networks/README.md).
+
+The EigenDA Disperser exposes 4 endpoints:
+
+1. `DisperseBlob()`
+2. `DisperseBlobAuthenticated()`
+3. `GetBlobStatus()`
+4. `RetrieveBlob()`
+
+These endpoints enable the blob lifecycle, from enqueuing blobs for dispersal to waiting for their dispersal finalization and finally to retrieving blobs from the EigenDA network. The following flowchart describes how move blobs through this lifecycle with respect to these endpoints:
+
+<!-- Flow chart pseudocode -->
+
+```mermaid
+graph EigenDA Disperser API Usage Flow;
+    A[Blob Ready]
+    B[Blob Queued for Dispersal]
+    C[Blob Dispersal Finalized]
+    D[Blob Expired]
+
+    A --> |DisperseBlob()| B
+    A --> |DisperseBlobAuthenticated()| B
+    B -->|GetBlobStatus() != finalized| B;
+    B -->|GetBlobStatus() == finalized| C;
+    C -->|RetrieveBlob()| C;
+    C -->|15 days elapses since dispersal finalization| D;
+```
+
+The Disperser offers an asynchrounous API for dispersing blobs, where clients should poll the GetBlobStatus() endpoint with the dispersal request ID they received from calling one of the two disperse endpoints until the disperser reports the blob as successfully dispersed and finalized.
+
+## Endpoints
+
+### Dispersal Endpoints
+
+There are two dispersal endpoints, `DisperseBlob()` and `DisperseBlobAuthenticated()`, which achive the same thing under different authentication strategies ( counterintuitively, both endpoints are authenticated in mainnet). Both endpoints accept a [DisperseBlobRequest](https://github.com/Layr-Labs/eigenda/blob/4d19b0597d1151a402e328430a0ff5345a831fb6/api/proto/disperser/disperser.proto#L72) message, with the following fields:
+
+:::warning
+It's important to note that during dispersal, blobs are automatically padded with zero bytes such that the final length of the blob in bytes is a multiple of 32. One implication of this is that a naive EigenDA client implementation will not retrieve the exact same blob that was dispersed.
+:::
+
+:::warning
+Every 32 bytes of data chunk is interpreted as an integer in big endian format where the lower address has more significant bits. The integer must stay in the valid range to be interpreted as a field element on the bn254 curve. The valid range is:
```

Comment:
Would it be better to just link to the page? This is seems like maybe redundant/duplication?